scripts/mktcli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env babel-node
+
+import { eth, Log, cli, contracts } from 'decentraland-commons'
+import { loadEnv } from './utils'
+import { decodeAssetId, encodeAssetId } from './monitor/utils'
+
+const log = new Log('mktcli')
+
+const logError = err => log.error('ERR: ' + err.message)
+const parseCoords = coords =>
+  coords
+    .replace('(', '')
+    .replace(')', '')
+    .split(',')
+
+const landRegistry = () => eth.getContract('LANDRegistry')
+const marketplace = () => eth.getContract('Marketplace')
+
+const main = {
+  addCommands(program) {
+    program.command('decode').action(async options => {
+      if (!options.length) return
+
+      try {
+        const coords = await decodeAssetId(options)
+        log.info(`(decode) str:${options} => coords:(${coords})`)
+      } catch (err) {
+        logError(err)
+      }
+    })
+
+    program.command('encode').action(async options => {
```

Comment:
_This goes for each command that receives a parameter_
I think we should add each required parameter as an explicit parameter of the command like we do here: https://github.com/decentraland/marketplace/blob/master/scripts/seed.js#L17 so, for example the help command works correctly:

![image](https://user-images.githubusercontent.com/692077/37898834-256c6ce8-30ea-11e8-9619-84a1262b81c4.png)